Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -0,0 +1,603 @@
+                paymentMethodTypes: paymentMethodTypes,
+                confirmHandler: confirmHandler
+            )
+        case .setup:
+            return PaymentSheet.IntentConfiguration(
+                mode: .setup(currency: settings.currency.rawValue, setupFutureUsage: .offSession),
+                paymentMethodTypes: paymentMethodTypes,
+                confirmHandler: confirmHandler
+            )
+        }
+    }
+
+    var clientSecret: String?
+    var ephemeralKey: String?
+    var customerID: String?
+    var paymentMethodTypes: [String]?
+    var amount: Int?
+    var checkoutEndpoint: String = PaymentSheetTestPlaygroundSettings.defaultCheckoutEndpoint
+    var addressViewController: AddressViewController?
+    var appearance = PaymentSheet.Appearance.default
+
+    func makeAlertController() -> UIAlertController {
+        let alertController = UIAlertController(
+            title: "Complete", message: "Completed", preferredStyle: .alert)
+        let OKAction = UIAlertAction(title: "OK", style: .default) { (_) in
+            alertController.dismiss(animated: true, completion: nil)
+        }
+        alertController.addAction(OKAction)
+        return alertController
+    }
+
+    private var subscribers: Set<AnyCancellable> = []
+
+    init(settings: PaymentSheetTestPlaygroundSettings) {
+        // Enable experimental payment methods.
+        //        PaymentSheet.supportedPaymentMethods += [.link]
+        PaymentSheet.enableACHV2InDeferredFlow = true // TODO(https://jira.corp.stripe.com/browse/BANKCON-6731) Remove this.
+
+        // Hack to ensure we don't force the native flow unless we're in a UI test
+        if ProcessInfo.processInfo.environment["UITesting"] == nil {
+            UserDefaults.standard.removeObject(forKey: "FINANCIAL_CONNECTIONS_EXAMPLE_APP_ENABLE_NATIVE")
+        } else {
+            // This makes the Financial Connections SDK use the native UI instead of webview. Native is much easier to test.
+            UserDefaults.standard.set(true, forKey: "FINANCIAL_CONNECTIONS_EXAMPLE_APP_ENABLE_NATIVE")
+        }
+        self.settings = settings
+
+        $settings.sink { _ in
+            self.load()
+        }.store(in: &subscribers)
+    }
+
+    func buildPaymentSheet() {
+        let mc: PaymentSheet
+
+        switch self.settings.integrationType {
+        case .normal:
+            switch self.settings.mode {
+            case .payment, .paymentWithSetup:
+                mc = PaymentSheet(paymentIntentClientSecret: self.clientSecret!, configuration: configuration)
+            case .setup:
+                mc = PaymentSheet(setupIntentClientSecret: self.clientSecret!, configuration: configuration)
+            }
+        case .deferred_csc, .deferred_ssc, .deferred_mp, .deferred_mc:
+            mc = PaymentSheet(intentConfiguration: intentConfig, configuration: configuration)
+        }
+
+        self.paymentSheet = mc
+    }
+
+        func didTapShippingAddressButton() {
```

Comment:
spacing of all these functions looks off.

Reply:
I was hoping the linter would fix this, blurgh